### ISSUE TITLE
Ticking Array out of bounds - Stagnant Water potion effect

### ISSUE THREAD

**Issue by MidnightsFX** [Angry-Pixel/The-Betweenlands#295]
When stepping into Stagnant water in the wight fortress I get this array out of bounds ticking exception.

```

Description: Ticking entity

java.lang.ArrayIndexOutOfBoundsException: -15
	at net.minecraft.potion.PotionEffect.func_76455_a(PotionEffect.java:105)
	at net.minecraft.entity.EntityLivingBase.func_70679_bo(EntityLivingBase.java:524)
	at net.minecraft.entity.EntityLivingBase.func_70030_z(EntityLivingBase.java:293)
	at net.minecraft.entity.Entity.func_70071_h_(Entity.java:318)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1561)
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:288)
	at net.minecraft.client.entity.EntityClientPlayerMP.func_70071_h_(SourceFile:63)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2006)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.potion.PotionEffect.func_76455_a(PotionEffect.java:105)
	at net.minecraft.entity.EntityLivingBase.func_70679_bo(EntityLivingBase.java:524)
	at net.minecraft.entity.EntityLivingBase.func_70030_z(EntityLivingBase.java:293)
	at net.minecraft.entity.Entity.func_70071_h_(Entity.java:318)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1561)
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:288)
	at net.minecraft.client.entity.EntityClientPlayerMP.func_70071_h_(SourceFile:63)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at net.minecraft.world.World.func_72939_s(World.java:1887)

-- Entity being ticked --
Details:
	Entity Type: null (net.minecraft.client.entity.EntityClientPlayerMP)
	Entity ID: 470
	Entity Name: MidnightsFX
	Entity's Exact location: -799.50, 82.54, -512.08
	Entity's Block location: World: (-800,82,-513), Chunk: (at 0,5,15 in -50,-33; contains blocks -800,0,-528 to -785,255,-513), Region: (-2,-2; contains chunks -64,-64 to -33,-33, blocks -1024,0,-1024 to -513,255,-513)
	Entity's Momentum: -0.08, -0.16, -0.01
Stacktrace:
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2006)

```

As I stated in the other ticket, I do have other mods installed and I can provide a stripped down test of this if needed, although it does look like this relates directly to this mod. I do not have conflicting potion IDs. If you need anymore information I would be glad to provide it!

**Comment by TheCyberBrick** [comment]
This was fixed in 8fa1ce2fbffa7d037c9b9138aa1cde59716e1212

**Comment by MidnightsFX** [comment]
Awesome! Amazing work on the mod by the way!
